lib/rest_6.js

```diff
@@ -181,24 +236,29 @@ function* getContractString(name, filename) {
  * @param{String} methodName the target method
  * @param{Object} args the arguments to be supplied to the targer method
  * @param{Number} value
+ * @param{Object} options flags for history and indexing
  * @param{Boolean} doNotResolve tell bloc to wait for resolution or return immediately
  * @param{Number} node target node
  * @returns{()} doNotResolve=true: [transaction hash] (String), doNotResolve=false: [method call return vals] (String|Int)
  */
 
-function* call(user, contract, methodName, args, value, doNotResolve, chainId, node) {
+function* call(user, contract, methodName, args, value, options, doNotResolve, chainId, node) {
```

Comment:
This will break all the existing applications. Can we place the options at the end. Should we discuss it?